examples/kafka_reader/kafka_reader.go

```diff
@@ -22,7 +23,7 @@ func main() {
 
 	brokerList := strings.Split(*brokers, ",")
 
-	k := kafka.New(brokerList, *topic)
+	k := kafka.New(brokerList, *group, *topic)
```

Comment:
for  examples/kafka_reader/kafka_reader.go  
```
group   = flag.String("group", filepath.Base(os.Args[0]), "the consumer group name")
k := kafka.New(brokerList, *topic) 
k.Group = *group
```
if driver code not provide a group arg, It could be a little bit of a problem.